source/src/core/io/raw_data/ScoreFileData.hh

```diff
@@ -42,11 +41,19 @@ public:
 		std::map < std::string, std::string > const & string_map = ( std::map < std::string, std::string > () )
 	);
 
-	bool write_pose(
-		const core::pose::Pose & pose,
+	/// @brief write the given data to a scorefile
+	bool write_scorefile(
+		std::string tag,
 		std::map < std::string, core::Real > const & score_map,
+		std::map < std::string, std::string > const & string_map,
+		bool use_json
+	);
+
+	/// @brief write the given data to a scorefile, autodetecting the format.
```

Comment:
"autodetecting the format" sounds a bit misleading to me, - maybe "use command line option <OPTION name> to determine output format".